.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
```

Comment:
Back when I still used Windows, I used to add this to my git config: `git config --global core.autocrlf true`
Does that do the same thing?
(Except for the fact that `git config` pertains to my local machine and this file pertains to this repository)

Reply:
No, they are not quite the same. They work in tandem, though.
According to the docs: https://git-scm.com/docs/gitattributes#_end_of_line_conversion

> While Git normally leaves file contents alone, it can be configured to normalize line endings to LF in the repository and, optionally, to convert them to CRLF when files are checked out.
> 
> If you simply want to have CRLF line endings in your working directory regardless of the repository you are working with, you can set the config variable "core.autocrlf" without using any attributes.
> ```ini
> [core]
> 	autocrlf = true
> ```
> This does not force normalization of text files, but does ensure that text files that you introduce to the repository have their line endings normalized to LF when they are added, and that files that are already normalized in the repository stay normalized.
> 
> If you want to ensure that text files that any contributor introduces to the repository have their line endings normalized, you can set the text attribute to "auto" for all files.
> ```txt
> *	text=auto
> ``` 

So, the attributes are basically a more granular EOL-setting for all users.

Reply:
See rebased version.  
I experimented with it on my machine and it works as expected.   
At least for newly checked-out repos all files are LF coming in. For some reason `git add --renormalize .` didn't really work. Not sure if this is a git bug or whatever. But in the end it works fine.

Reply:
I don't think there are any Windows users who are keeping a long-running checkout of this project, so we're probably fine :)